cdrom/aulas/static/aulas/css/main.css

```diff
@@ -1,3 +1,10 @@
+.body{
```

Comment:
```suggestion
body{
```
Usar el objeto body, no necesitamos una clase